rootfs/etc/profile.d/_20-mounts.sh

```
@@ -0,0 +1,206 @@
+		if [[ -e $dest ]]; then
+			red "# ERROR: Symlink destination already exists: '$dest'"
+			continue
+		fi
+		ln -sT "$src" "$dest" && yellow symlinking "'$src' -> '$dest'" || red "# ERROR: Failed to create symlink: '$src' -> '$dest'"
+	done
+}
+
+# Ensure we only run this once per container
+if [[ -f "${GEODESIC_HOST_PATHS_CACHE:=/tmp/geodesic-host-paths}" ]]; then
+	# Source the cached paths
+	. "${GEODESIC_HOST_PATHS_CACHE}"
```

Shellcheck (0.10.0)

[warning] ShellCheck can't follow non-constant source. Use a directive to specify location.

(SC1090)